Controller/PasswordReset/ResetRequestController.php

```diff
@@ -93,15 +93,18 @@ public function __construct(
     public function requestAction(Request $request)
     {
         $form = $this->formFactory->create($this->formType);
-        $form->handleRequest($request);
 
-        if ($form->isValid()) {
+        if ($form->handleRequest($request)->isSubmitted() && $form->isValid()) {
 
             $user = $this->getUser($form);
             if (!($user instanceof ResettablePasswordInterface)) {
                 return $this->addFlashAndRedirect('info', 'admin.password_reset.request.mail_sent_if_correct');
             }
 
+            if (!$user->isEnabled()) {
+                return $this->addFlashAndRedirect('warning', 'admin.password_reset.request.user_disabled');
```

Comment:
this is a security issue as long as it allows to detect user existence. this information should be sent via e-mail :)